app/Http/Controllers/Search/MiscController.php

```diff
@@ -353,4 +353,15 @@ protected function buildSearchResults($term, $class, array $excludes = [])
 
         return Response::json($formatted);
     }
+
+    /**
+     * @param Request $request
+     * @return mixed
+     */
+    public function questElements(Request $request)
```

Comment:
Not needed, we already have this with filtering on some entity types which can't be in quests